java/lance-catalog-apache-client/api/openapi.yaml

```diff
@@ -46,6 +46,192 @@ paths:
       summary: Ping the server for use cases like health check
       x-accepts:
       - application/json
+  /v1/namespaces:
```

Comment:
@jackye1995 Must we have a mirror version of the spec?

Reply:
sorry what do you mean by mirror version?

Reply:
I mean, this is a copied version of `spec/catalog.yaml`?

Reply:
oh I see. Yeah there are quite a few things that can be omitted. I had an initial list in `.apache-client-ignore` and we should add more to it. I think I originally had that `api` folder in list, but there were some issues in general about the syntax to ignore an entire folder, I did not figure out the right way to express that it yet last time. If you know you can also update that and rerun `make gen`

Reply:
Did some more experiments, I added the following and the unnecessary directories are no longer generated:

```
**/gradle/**
**/.github/**
**/api/**
```